src/main/java/com/yahoo/bullet/pubsub/PubSubMessage.java

```diff
@@ -0,0 +1,87 @@
+package com.yahoo.bullet.pubsub;
+
+import java.io.Serializable;
+import com.yahoo.bullet.pubsub.Metadata.Signal;
+import lombok.Getter;
+
+@Getter
+public class PubSubMessage implements Serializable {
+    private String id;
+    private String content;
+    private Metadata metadata;
+    private long sequenceNumber;
```

Comment:
Use int, call it just sequence (it's clear it's a number) and can you move this to right after id?